app/src/main/res/values-fr/strings.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="app_label" translatable="false">Moonlight</string>
+    <string name="app_label_root" translatable="false">Moonlight (Root)</string>
```

Comment:
These should be removed, since they're not translatable.

Reply:
Okay no worries thank you